src/app/owners/owner-add/owner-add.component.ts

```diff
@@ -32,6 +33,7 @@ import {Router} from '@angular/router';
 })
 export class OwnerAddComponent implements OnInit {
 
+  @ViewChild("ownerForm", { static: true }) ownerForm: NgForm;
```

Comment:
Why did you introduced the `@ViewChild`decorator?

Reply:
it was throwing error as ownerForm not found in component ts file